easybuild/easyconfigs/q/Qt/Qt-4.8.6-intel-2015a.eb

```diff
@@ -1,19 +1,32 @@
+#easyblock = 'CMakeMake'
+
 name = 'Qt'
 version = '4.8.6'
 
 homepage = 'http://qt-project.org/'
 description = "Qt is a comprehensive cross-platform C++ application framework."
 
-toolchain = {'name': 'intel', 'version': '2015a'}
+toolchain = {'name': 'intel', 'version': '2015a'} 
 
-source_urls = [
-    'http://origin.releases.qt-project.org/qt4/source/',
-    'http://master.qt-project.org/archive/qt/%(version_major_minor)s/%(version)s/'
-]
+source_urls = ['http://download.qt-project.org/official_releases/qt/4.8/4.8.6/']
 sources = ['%(namelower)s-everywhere-opensource-src-%(version)s.tar.gz']
 
-patches = ['Qt-4.8.6_icc-qUnused.patch']
 
-dependencies = [('GLib', '2.40.0')]
+dependencies = [
+    ('GLib', '2.42.1'),
+    ('Mesa', '7.11.2', '-Python-2.7.9'),
```

Comment:
this Mesa version is quite old... I have an *almost* working newer version of Mesa in #1226, but I got stuck since it requires libudev, and I haven't figured out yet how to flesh that out of the systemd source tarball it's now shipped in (yes, ...)